### PR TITLE
F13/register page

### PR DESCRIPTION
**Pull request by M-Kulinkovich**
There are errors, I solve them, filled in so that they looked
;)

### CODE REVIEW

File: app/src/components/signIn/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            </div>
+            <div>
+              <span>Test 2</span>
+            </div>
+            <div>
+              <span>Test 3</span>
+             </div>
+            <div>
+              <span>Что-то прекрасное о вузе</span>
+            </div>
```
Comment (iruseg00) [inline app/src/components/signIn/SignIn.js]: Те же ошибки что и в LoginPage , ток еще и с табуляцией проблемы...

File: app/src/App.js [hunk 2]
```diff
@@ -16,7 +17,8 @@ function App() {
       <Route path="/calendar" component={Menu} />
       <Route path="/search" component={Menu} />
       <Route path="/login" component={Login} />
-      <Route path="/dashboard" component={Dashboard} />
+      <Route path="/signIn" component={SignIn} />
+       <Route path="/dashboard" component={Dashboard} />
```
Comment (Billerens) [inline app/src/App.js]: We need pure code, correct your tabulations

File: app/src/components/signIn/style.module.scss [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  height: 100%;
+}
+.bar {
+  height: 100%;
+  width: $bar-width;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background: black;
```
Comment (Billerens) [inline app/src/components/signIn/style.module.scss]: To directives

File: app/src/components/signIn/SignIn.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+          label={
+            <span>
+              Nickname&nbsp;
+              <Tooltip title="What do you want others to call you?">
+                <Icon type="question-circle-o" />
+              </Tooltip>
+            </span>
+          }
+        >
+          {getFieldDecorator('nickname', {
```
Comment (Billerens) [inline app/src/components/signIn/SignIn.js]: Not "nickname", but "username"

File: app/src/components/signIn/SignIn.js [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+            rules: [{ required: true, message: 'Please input your nickname!', whitespace: true }],
```
Comment (Billerens) [inline app/src/components/signIn/SignIn.js]: here too

File: app/src/components/signIn/SignIn.js [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+            Register
+          </Button>
+        </Form.Item>
+      </Form>
+    );
+  }
+}
+</div>  
+</div> 
+</div>
```
Comment (Billerens) [inline app/src/components/signIn/SignIn.js]: Correct your tabulations

**Pull request merged**